src/main/java/me/arminb/sara/controller/UserController.java

```diff
@@ -74,38 +63,46 @@
     public ResponseEntity<User> addUser(@RequestBody UserRequest userReq) {
         try {
             User user = userReq.toUser();
-            return new ResponseEntity<User>(userService.save(user), HttpStatus.OK);
+            return new ResponseEntity<User>(userService.saveUser(user), HttpStatus.OK);
```

Comment:
why did you change the method name to saveUser? I would revert it to save again.